subt_ign/src/GameLogicPlugin.cc

```diff
@@ -1375,6 +1402,13 @@ bool GameLogicPluginPrivate::OnPoseFromArtifact(
   return result;
 }
 
+/////////////////////////////////////////////////
+void GameLogicPluginPrivate::LogEvent(const std::string &_event)
+{
+  std::lock_guard<std::mutex> lock(this->eventMutex);
+  this->eventStream << _event;
```

Comment:
add `this->eventStream.flush();` here?

Reply:
Added: 762e1dd713a9e608ed1e4b302deb8273df15fc1d